web/src/pygments_data.ts

```diff
@@ -0,0 +1,7 @@
+import generated_pygments_data from "../generated/pygments_data.json";
+
+const langs: Record<string, {priority: number; pretty_name: string} | undefined> = generated_pygments_data.langs;
+
+export {
+    langs,
+}
```

Comment:
Fix the missing newline at end-of-file here.

Reply:
Done, thanks. (Does the linter not catch this? Or maybe I was in one of those situations where the linter doesn't run on newly added files in some situations.)